selfdrive/ui/qt/offroad/settings.cc

```diff
@@ -214,20 +214,39 @@ void DeveloperPanel::showEvent(QShowEvent *event) {
   Params params = Params();
   std::string brand = params.getBool("Passive") ? "dashcam" : "openpilot";
   QList<QPair<QString, std::string>> dev_params = {
-    {"Version", brand + " v" + params.get("Version", false).substr(0, 14)},
     {"Git Branch", params.get("GitBranch", false)},
     {"Git Commit", params.get("GitCommit", false).substr(0, 10)},
     {"Panda Firmware", params.get("PandaFirmwareHex", false)},
     {"OS Version", Hardware::get_os_version()},
   };
 
+  std::string lastUpdateTimeRaw = params.get("LastUpdateTime", false);
+  QString version = QString::fromStdString(brand + " v" + params.get("Version", false).substr(0, 14)).trimmed();
+  QString lastUpdateTime = QString::fromStdString(lastUpdateTimeRaw.substr(0,10) + " at " + lastUpdateTimeRaw.substr(12,7)).trimmed();
```

Comment:
is there some nice Qt function to get us a nicely formatted time like "one day ago"?

Reply:
I wasn't able to find any Qt functions for this exact format ("one day ago"), but I used the QDateTime::daysTo() to make something similar

Reply:
let's make a function in `selfdrive/ui/qt/util.h` that returns the string given a QDateTime. it should output:

Reply:
Done. I also added   x hours ago, when < 1 day